src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Record.txt

```diff
@@ -15,8 +15,11 @@ false
 >> IsBlank({ x : Blank(), y : 2}.x)
 true
 
->> First([{a: 1}]).Value.a
+>> First([{a: 1}]).a
```

Comment:
We need to decide how we're managing the breaking change here. IE, these tests are shared by Canvas too; so they'll run against JS (which does not yet have this update; and it can't have it until PowerApps has a converter).